Backend/Monolithic/Startup.cs

```diff
@@ -92,16 +84,26 @@ public void ConfigureServices(IServiceCollection services)
                     options.SerializerSettings.Formatting = Formatting.Indented;
                 });
 
-            // Configure the Swagger generator
-            services.AddSwaggerGen(c =>
-            {
-                c.SwaggerDoc("1.0", new Info { Title = "Contoso Maintenance API", Version = "1.0" });
+            // Configure the Swagger generator            
+            services.AddSwaggerGen(this.SwaggerGen("Job", "api/job"));
+            services.AddSwaggerGen(this.SwaggerGen("Part", "api/part"));
+            services.AddSwaggerGen(this.SwaggerGen("Dummy", "api/dummy"));
+            services.AddSwaggerGen(this.SwaggerGen("Photo", "api/photo"));
+            services.AddSwaggerGen(this.SwaggerGen("Search", "api/search"));
+        }
 
+        private Action<SwaggerGenOptions> SwaggerGen(string name, string filter)
+        {
+            return c =>
+            {
+                c.SwaggerDoc(name, new Info { Title = $"Contoso Maintenance API - {name}", Version = "1.0" });
+                
+                c.DocumentFilter<SwaggerFilter>(name, filter);

```

Comment:
Same here, please explain in a comment, what the `SwaggerFilter` does. Especially what is does here.